Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -291,7 +291,52 @@ script:
     description: Creates a new channel in a Microsoft Teams team.
     execution: false
     name: microsoft-teams-create-channel
-  dockerimage: demisto/teams:1.0.0.13080
+  - arguments:
+    - default: false
+      description: The meeting start time.
+      isArray: false
+      name: start_date_time
+      required: true
+      secret: false
+    - default: false
+      description: The meeting end time.
```

Comment:
please elaborate on the date format and add example
also, why is this mandatory ?